README.md

```diff
@@ -26,12 +27,19 @@ version of the subgraph to IPFS or a local directory for debugging.
 Subgraphs for The Graph are set up just like any regular TypeScript
 project. It is recommended to install `graph-cli` as a local dependency
 via `package.json` and use `npm` scripts for code generation and
-building.
+building. <br/>_
 
 If you are just getting started with creating a subgraph, head to [getting started](https://github.com/graphprotocol/graph-node/blob/master/docs/getting-started.md). Eventually this guide will lead you back here.
 
 For clarity, an example of the setup below can be found in the [ENS subgraph repository](https://github.com/graphprotocol/ens-subgraph).
 
+### On Linux
+`libsecret` is used for storing auth tokens, so you
+may need to install it before getting started. Use one of the following commands depending on your distribution:
```

Comment:
The newline here looks odd. Suggestion: balance the paragraph more evenly, like so
```
libsecret is used for storing auth tokens, so you may need to install it
before getting started. Use one of the following commands depending on
your distribution:
```